alfalfa_worker/requirements/production.txt

```diff
@@ -14,7 +14,8 @@ jmespath==0.9.4
 mlep==0.1.dev2
 more-itertools==5.0.0
 nodeenv==1.3.4
-pandas==0.24.2
+numpy==1.16.6  # need older version until Python > 3.5
```

Comment:
need to upgrade to > Python 3.5. However, we can't do this until we upgrade away from Ubuntu 16.0x and ideally the dual python versions.